webanno-ui-annotation/src/main/java/de/tudarmstadt/ukp/clarin/webanno/ui/annotation/AnnotationPage.java

```diff
@@ -334,7 +334,7 @@ protected CAS onGetJCas() throws IOException
             }
 
             @Override
-            protected void onJcasUpdate(Long aTimeStamp)
+            protected void onJCasUpdate(Long aTimeStamp)
```

Comment:
Please submit generic changes outside the Codebook module as separate Issues/PRs against master.

Reply:
Never mind, I see this method doesn't even exist in master.

Instead of adding this method, you could consider listening to `AnnotationEvent` or `AfterCasWrittenEvent`.

Reply:
Thanks for this tip! I just had a look into this possibility and will most probably use this in future